pkg/rancher-desktop/pages/containers/ContainerInfo.vue

```diff
@@ -0,0 +1,386 @@
+    },
+    currentContainer() {
+      if (!this.containers || !this.containerId) {
+        return null;
+      }
+      return this.containers[this.containerId];
+    },
+    containerName() {
+      if (!this.currentContainer) {
+        return this.containerId.substring(0, 12);
+      }
+      const names = this.currentContainer.names || this.currentContainer.containerName;
+      if (Array.isArray(names)) {
+        return names[0]?.replace(/^\//, '').replace(/_[a-z0-9-]{36}_[0-9]+/, '') || this.containerId.substring(0, 12);
+      }
+      if (typeof names === 'string') {
+        return names.replace(/^\//, '').replace(/_[a-z0-9-]{36}_[0-9]+/, '') || this.containerId.substring(0, 12);
+      }
+      return this.containerId.substring(0, 12);
+    },
+    containerState() {
+      if (!this.currentContainer) {
+        return 'unknown';
+      }
+      return this.currentContainer.state || this.currentContainer.status || 'unknown';
+    },
+    isRunning() {
+      if (!this.currentContainer) {
+        return false;
+      }
+      return this.currentContainer.state === 'running' || this.currentContainer.status === 'Up';
+    },
+  },
+  watch: {
+    containerName: {
+      handler(name) {
+        this.$store.dispatch('page/setHeader', {
+          title:       name || 'Container Info',
+          description: '',
+          action:      'ContainerStatusBadge',
+        });
+      },
+      immediate: true,
+    },
+  },
+  mounted() {
+    ipcRenderer.on('settings-read', (event, settings) => {
+      this.settings = settings;
+      this.subscribe().catch(console.error);
+    });
+
+    ipcRenderer.send('settings-read');
+
+    ipcRenderer.on('settings-update', (_event, settings) => {
+      this.settings = settings;
+    });
+
+    this.subscribe().catch(console.error);
+
+    window.addEventListener('keydown', this.handleGlobalKeydown);
+  },
+  beforeUnmount() {
+    this.$store.dispatch('page/setHeader', { action: null });
+    ipcRenderer.removeAllListeners('settings-update');
+    ipcRenderer.removeAllListeners('settings-read');
+    this.$store.dispatch('container-engine/unsubscribe').catch(console.error);
+    clearTimeout(this.subscribeTimer);
+    window.removeEventListener('keydown', this.handleGlobalKeydown);
+  },
+  methods: {
+    async subscribe() {
+      clearTimeout(this.subscribeTimer);
+      try {
+        if (!window.ddClient || !this.isK8sReady || !this.settings) {
+          this.subscribeTimer = setTimeout(() => this.subscribe(), 1_000);
+          return;
+        }
+        await this.$store.dispatch('container-engine/subscribe', {
+          type:   'containers',
+          client: window.ddClient,
+        });
+      } catch (error) {
+        console.error('There was a problem subscribing to container events:', { error });
+      }
+    },
+    onSearchInput() {
+      if (this.$refs.containerLogs) {
+        this.$refs.containerLogs.performSearch(this.searchTerm);
+      }
```

Comment:
```suggestion
      this.$refs.containerLogs?.performSearch(this.searchTerm);
```

I would suggest making `this.containerLogs` a computed getter so it can have the correct type information too.  Same with `searchInput`.